server.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const session = require('express-session');
+
+const User = require('./userModel');
+
+mongoose
+    .connect('mongodb://localhost/auth-i')
+    .then(() => { console.log('\n*** Connected to database ***\n'); })
+    .catch(error => console.log('\n!!!Error connecting to DB!!!\n', error));
+
+const server = express();
+
+const sessionOptions = {
+    secret: ' Space the final frontier ',
+    cookie: {
+        maxAge: 1000 * 60 * 60 * 24 // a day
+    },
+    httpOnly: true,
+    secure: false,
+    resave: true,
+    saveUninitialized: false,
+    name: 'noname',
+};
+
+function protected(req, res, next) {
+    if (req.session && req.session.username) {
+        next();
+    } else {
+        res.status(401).json({ message: 'You shall not pass!' });
+    }
+};
+
+server.use(express.json());
+server.use(session(sessionOptions));
+
+// server.get('/', (req, res) => {
+//     res.status(201).json('Server Running')
+// });
+
+server.get('/', function (req, res) {
+    if (req.session && req.session.username) {
+        res.status(200).json({ message: `Welcome back ${req.session.username}` });
+    } else {
+        res.status(401).json({ message: 'Speak friend and enter' });
+    }
+});
+
+// server.post('/api/register', (req, res) => {
```

Comment:
I love the practice of commenting iterations as it tells a story but this is a 1:1 duplicate haha
Doesn't effect anything just found it odd.

Reply:
Those are of day 1 / day 2 comparison note for me.
Commit history was not enough haha